doc/playbook.rst

```diff
@@ -127,6 +127,15 @@ It accepts messages and gateways them to Cloud Run.
 tester
 ======
 
+Run the tester either on a local machine, or in Cloud Shell.  
```

Comment:
I think the text defining what the tester _is_ really should go first; it's not a very intuitive name. I suggest moving the new text between "It takes a JSON token..." and "Command line arguments are...".